src/main/java/games/battlelore/BattleloreGameState.java

```diff
@@ -0,0 +1,264 @@
+package games.battlelore;
+
+import core.AbstractGameState;
+import core.AbstractParameters;
+import core.components.Component;
+import core.components.Deck;
+import core.components.GridBoard;
+import core.components.PartialObservableDeck;
+import core.interfaces.IGamePhase;
+import core.interfaces.IStateHeuristic;
+import core.turnorders.TurnOrder;
+import games.GameType;
+import games.battlelore.cards.CommandCard;
+import games.battlelore.components.MapTile;
+import games.battlelore.components.Unit;
+
+import java.util.*;
+
+public class BattleloreGameState extends AbstractGameState {
+
+    public enum BattleloreGamePhase implements IGamePhase {
+        CommandAndOrderStep, //Player Plays One Command Card
+        MoveStep,
+        AttackStep,
+        VictoryPointStep,
+        DrawStep,
+        LoreStep
+        //Last three steps can be implemented later
+    }
+
+    public enum UnitType {
+        Decoy, BloodHarvester, ViperLegion, CitadelGuard, YeomanArcher;
+    }
+
+    int numberOfRounds;
+    int[] playerScores;
+    GridBoard<MapTile> gameBoard;
+    List<Unit> unitTypes;
+    BattleloreGameParameters parameters;
```

Comment:
Re-adding comment: There is already a parameter variable in the super class, should just use that.